lib/Perl/Critic/Policy/Moose/RequireCleanNamespace.pm

```diff
@@ -44,6 +44,12 @@ sub violates {
     return if not $includes;
 
     for my $include ( @{$includes} ) {
+        # skip if nothing imported
+        if ( $include->type eq 'use' ) {
+            my $lists = $include->find('PPI::Structure::List');
+            next if $lists && @$lists == 1 && $lists->[0]->children == 0;
```

Comment:
Arguably this should be `!grep { $lists->[$_]->children > 0 } $#{$lists}`, to allow for `use Moose () ()`, but that's getting *really* pedantic :D